client/gui_rpc_server_ops.cpp

```diff
@@ -1072,7 +1077,9 @@ static void read_all_projects_list_file(GUI_RPC_CONN& grc) {
     int retval = read_file_string(ALL_PROJECTS_LIST_FILENAME, s);
     if (!retval) {
         strip_whitespace(s);
-        grc.mfout.printf("%s\n", s.c_str());
+        const char *q = strstr(s.c_str(), "<projects>");
+        if (!q) q = "<projects/>";        
+        grc.mfout.printf("%s\n", q);
```

Comment:
This is more about the commit messages. I dislike a message like 'fix previous commit'. What does 'previous' mean in git? If I cherry-pick this commit 'previous' will get a complete new meaning. Please state why you changed what you changed in the commit. 
Please also prepend the first line with the component ("Client:"). This used to be an unwritten rule for every BOINC developer from the beginning but I just saw that even seasoned BOINC developers don't uphold this rule.